docs/tutorial/01_getting_started.md

```diff
@@ -0,0 +1,15 @@
+# 1. Getting Started
+
+## Bootstrap a Juju controller on AWS
+
+From your computer where you have installed Juju, add your AWS credentials to Juju:
```

Comment:
What version of Juju? Because later when bootstrapping EKS, it will be important.

Reply:
Done